hepcrawl/spiders/cds_spider.py

```diff
@@ -9,67 +9,53 @@
 
 """Spider for the CERN Document Server OAI-PMH interface"""
 
-from scrapy.spider import XMLFeedSpider
-from scrapy import Request
-from harvestingkit.inspire_cds_package.from_cds import CDS2Inspire
-from harvestingkit.bibrecord import (
-    create_record as create_bibrec,
-    record_xml_output,
-)
+import logging
+from flask.app import Flask
 from dojson.contrib.marc21.utils import create_record
 from inspire_dojson.hep import hep
 
-from . import StatefulSpider
+from .oaipmh_spider import OAIPMHSpider
 from ..utils import ParsedItem
 
+logger = logging.getLogger(__name__)
 
```

Comment:
two spaces between imports and globals and defs/class, and globals should be all caps.